bin/load_page_traffic.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+rake page_traffic:load
+
+SEARCH_INDEX=page-traffic rake search:clean
+
+SEARCH_INDEX=detailed PROCESS_ALL_DATA=true rake search:update_popularity
+sleep 2400
```

Comment:
Why the sleeps? (can we easily avoid them?)

Reply:
Sorry, good catch, I was meant re-visit this. They were in [the original script](https://github.com/alphagov/search-analytics/blob/d124229da53d9664512a74b47ded8dd50c27e8dc/nightly-run.sh#L28) that this is replacing.

Looking at the comment it seems each of the rake tasks queues a bunch of jobs. The [sleep attempts to prevent filling up Redis](https://github.com/alphagov/search-analytics/commit/8f24581b95af9b37581f16988f466364d5de5808)?

I think I could just call these as three cronjobs that run at intervals - don't think they necessarily need be guaranteed to run consecutively.

Reply:
Like this: https://github.com/alphagov/govuk-helm-charts/pull/1113